lib/web_ui/dev/utils.dart

```diff
@@ -62,6 +62,25 @@ Future<int> runProcess(
   return exitCode;
 }
 
+/// Runs [executable]. Do not follow the exit code or the output.
+void runProcessInTheBackground(
```

Comment:
A better name for this would be `startDetachedProcess` ("run" implies waiting until the process exits, and "detached" is the standard term used in the standard libraries rather than background).

Reply:
makes sense, thanks for the suggestion. done.